src/types/Integration.ts

```diff
@@ -7,12 +7,14 @@ export enum IntegrationType {
     WEBHOOK = 'webhook',
     EMAIL_SUBSCRIPTION = 'email_subscription',
     SPLUNK = 'camel:splunk',
-    SLACK = 'camel:slack'
+    SLACK = 'camel:slack',
+    SERVICE_NOW = 'camel:service_now'
```

Comment:
Can it be without the underscore? Either `servicenow` or `snow`.

Reply:
Sure, it can be `servicenow` - any particular reason you don't want the underscore? or just matter of preference?